client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -44,6 +44,7 @@ class SimplePaymentsEdit extends Component {
 
 	componentDidMount() {
 		this.injectPaymentAttributes();
+		this.validateAttributes();
```

Comment:
Should we do this on injection, as the PR name suggests? Or perhaps that's not really the problem we're trying to address:

https://github.com/Automattic/wp-calypso/blob/d1575dad23158f441059cbbf4b8a3a08a489c09b/client/gutenberg/extensions/simple-payments/edit.js#L72-L79